server/src/document/DbtTextDocument.ts

```diff
@@ -200,6 +200,12 @@ export class DbtTextDocument {
     }
   }
 
+  async resendDiagnostics(): Promise<void> {
+    if (this.destinationState.contextInitialized && this.dbt.dbtReady && !this.currentDbtError && !this.modelCompiler.compilationInProgress) {
```

Comment:
Why we do not need to send diagnostics if we have `this.currentDbtError` ? What if we will return to this doc after compile error in another one?

Reply:
You are right that we can send error here. But I decided to work on it in separate task since I'm still not sure that we should have this global error (according to feedback in slack).